src/lib/errors.js

```diff
@@ -43,13 +48,23 @@ class ErrorHandler {
    * @param {Function} restart callback
    * @param {Function} ignore callback
    */
-  toUser(error, errorLocation, restart, ignore) {
-    const errorString = `Error: ${errorLocation || "Unknown"}: ${error}`;
-    log.error(
-      errorString + (error.stack ? "\nstack trace: " + error.stack : "")
-    );
+  async toUser(error, errorLocation, restart, ignore) {
     if (window.getMain()) {
```

Comment:
Same here, reverse + return after `errorHandler.die` and the code gets a bit cleaner imo